### DISCUSSION TITLE
Threshold values given through postman

### DISCUSSION THREAD

**Discussion by george4joseph** [apideck-libraries/postman-to-k6#52] [Q&A]
@thim81 
To make completely automated,I would like to give thresholds through postman. Thresholds given to each request is postman to taken to converted k6-script and is used there as threshold. Is this idea is possible in postman-to-k6?

**Comment by thim81** [comment]
@george4joseph How can pass thresholds in Postman?

**Comment by george4joseph** [comment]
@thim81 

```
pm.variables.set('threshold_items',{
    "thresholds": {
      "http_req_duration{name:getUsers}": [
        {
          "threshold": "p(90) < 2000",
          "abortOnFail": true
        }
      ],
      "http_req_duration{name:getDevices}": ["p(90) < 900"],
      "http_req_duration{name:getUsersAS}": ["p(90) < 100"],
      "http_req_duration{group:::Groups}": ["p(90) < 300"]

    }
 
})
```
I pass it as pre script.. 
In the generated k6 script, if I could add it as 
`let options = threshold_items`    - it will work

**Reply by thim81** [reply]
it is currently not possible.

There is [open feature](https://github.com/apideck-libraries/postman-to-k6/issues/46 ) to introduce the option to pass a "setup()", which might be handy for your use-case. There is no timing for this feature, but if you need it urgently, feel free to create a PR.
